lib/indexer.rb

```diff
@@ -0,0 +1,121 @@
+module Argo
+  class Indexer
+    SimpleLogJob = Struct.new(:log_val) do
+      def perform
+        sleep(rand 5)
+        job_logger = Logger.new(File.open("tmp/simple_job.log"))
+        job_logger.unknown "#{log_val}"
+      end
+    end
+
+
+    def reindex_object obj
+      doc = obj.to_solr
+      Dor::SearchService.solr.add(doc)
+    end
+
+    def reindex_pid pid
+      obj = Dor.load_instance pid
+      reindex_object obj
+    end
+
+    def reindex_pid_list pid_list, should_commit=false
```

Comment:
Use def with parentheses when there are parameters.<br>Surrounding space missing in default value assignment.